AZ3166/src/libraries/AzureIoT/examples/RemoteMonitoringv2/RemoteMonitoring.ino

```diff
@@ -0,0 +1,226 @@
+  }
+  else
+  {
+    sprintf(wifiBuff, "WiFi  \r\n             \r\nNo connection\r\n                 \r\n");
+    Screen.print(wifiBuff);
+  }
+}
+
+void sendData(const char *data, const char *schema){
+  time_t t = time(NULL);
+  char buf[sizeof "2011-10-08T07:07:09Z"];
+  strftime(buf, sizeof buf, "%FT%TZ", gmtime(&t));
+
+  EVENT_INSTANCE* message = DevKitMQTTClient_Event_Generate(data, MESSAGE);
+
+  DevKitMQTTClient_Event_AddProp(message, "$$CreationTimeUtc", buf);
+  DevKitMQTTClient_Event_AddProp(message, "$$MessageSchema", schema);
+  DevKitMQTTClient_Event_AddProp(message, "$$ContentType", "JSON");
+  
+  DevKitMQTTClient_SendEventInstance(message);
+}
+
+void showSensors()
+{
+  try
+  {
+    ht_sensor->reset();
+    ht_sensor->getTemperature(&temperature);
+    ht_sensor->getHumidity(&humidity);
+
+    lp_sensor->getPressure(&pressure);
+    
+    char buff[128];
+    sprintf(buff, "Environment \r\n Temp:%sC    \r\n Humidity:%s%%  \r\n Atm: %spsig",f2s(temperature, 1), f2s(humidity, 1),f2s(pressure,1));
+    Screen.print(buff);
+
+    char sensorData[200];
+    sprintf_s(sensorData, sizeof(sensorData), "{\"temperature\":%s,\"temperature_unit\":\"%s\",\"humidity\":%s,\"humidity_unit\":\"%s\",\"pressure\":%s,\"pressure_unit\":\"%s\"}", f2s(temperature, 1), temperatureUnit,f2s(humidity, 1), humidityUnit,f2s(pressure, 1), pressureUnit);
+    sendData(sensorData,roomSchema);
+  }
+  catch(int error)
+  {
+    LogError("*** Read sensor failed: %d",error);
+  }
+}
+
+void setup() {
+  pinMode(LED_WIFI, OUTPUT);
+  pinMode(LED_AZURE, OUTPUT);
+  pinMode(LED_USER, OUTPUT);
+
+  ext_i2c = new DevI2C(D14, D15);
+  
+  ht_sensor = new HTS221Sensor(*ext_i2c);
+  ht_sensor->init(NULL);
+
+  lp_sensor= new LPS22HBSensor(*ext_i2c);
+  lp_sensor->init(NULL);
+  
+  //Scan networks and print them into console
+  int numSsid = WiFi.scanNetworks();
+  for (int thisNet = 0; thisNet < numSsid; thisNet++) {
+     Serial.print(thisNet);
+     Serial.print(") ");
+     Serial.print(WiFi.SSID(thisNet));
+     Serial.print("\tSignal: ");
+     Serial.print(WiFi.RSSI(thisNet));
+     Serial.print("\tEnc type: ");
+     Serial.println(WiFi.encryptionType(thisNet));
+  }   
+
+  InitWiFi();
+
+  if (isConnected)
+  {
+    // Microsoft collects data to operate effectively and provide you the best experiences with our products. 
+    // We collect data about the features you use, how often you use them, and how you use them.
+    send_telemetry_data_async("", "RemoteMonitoringSetup", "");
+  }
+}
+
+
+
+static uint64_t send_interval_ms;
+static uint64_t reset_interval_ms;
+static uint64_t blink_interval_ms;
+void loop() {
+  // put your main code here, to run repeatedly:
+  if(isConnected)
+  {
+    if((int)(SystemTickCounterRead() - send_interval_ms)>INTERVAL)
+    {
+      showSensors();
+      send_interval_ms = SystemTickCounterRead();
+    }
+
+    if((int)(SystemTickCounterRead() - blink_interval_ms)>500){
+      int value= digitalRead(LED_AZURE);
```

Comment:
LED_AZURE is defined to check whether the connection to Azure is succeed. It is auto turned on/off when MQTT client connect/disconnect to Azure. Please use LED_USER instead.